conda_build/pypi.py

```diff
@@ -86,7 +86,8 @@
   # You can also put a file called run_test.py in the recipe that will be run
   # at test time.
 
-  # requires:
+  requires:
+    - {tests_require}
```

Comment:
The way `INDENT` works, it already includes the leading spaces and `-`. I think this should be

```
  {requires_comment}requires:{tests_require}
```

where `requires_comment` is `# ` if there are no test requires and empty if there are some.